src/textual/message_pump.py

```
@@ -274,7 +274,11 @@ async def _process_messages(self) -> None:
                         for _cls, method in self._get_dispatch_methods(
                             "on_idle", event
                         ):
-                            await invoke(method, event)
+                            try:
+                                await invoke(method, event)
```

This was being silently ignored. PITA to track down.